packages/bloc/lib/src/emitter.dart

```diff
@@ -102,6 +102,7 @@ class _Emitter<State> implements Emitter<State> {
       onData: (data) => call(onData(data)),
       onError: onError != null
           ? (Object error, StackTrace stackTrace) {
+              addError(error, stackTrace);
```

Comment:
Can you provide some more context regarding why you want this change? Typically you use `addError` to notify an observer of an unhandled error. In This case, the error is handled because an `onError` callback is provided so I don't think it would be consistent to automatically call `addError` here.